draft-ietf-mmusic-sdp-bundle-negotiation.xml

```diff
@@ -1421,6 +1421,12 @@ SDP Answer
       </t>
     </section>
 
+    <section title="RTP Header Extensions Consideration" anchor="sec-extmap" toc="default">
+      <t>When <xref target="RFC5285"/> RTP header extensions are used in the context of this 
+       specification, the identifier used for a given extension MUST identify the the same 

```

Comment:
It may be useful to mention "extmap" by name, to make it easy to find if someone searches for it. Could say "the identifier used for a given extension **in an 'a=extmap' attribute** MUST..."